### PR TITLE
Update VS Code extension overview to reflect Open Preview of Config Test Run

### PR DESCRIPTION
**Pull request by BeFunes**
## Please do not merge this before confirming with @BeFunes  (Benny) , likely January 6th.

# Description
Changed Config Test Run section to reflect planned changes to Closed Preview package:
- Feature will be opt-out instead of opt-in
- Performance customers will be included in the preview

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (rosieyohannan) [inline jekyll/_cci2/vs-code-extension-overview.adoc:205]:
```suggestion
You can opt-out of this feature per project. For your project, navigate to menu:Project Settings[VS Code integration] and toggle the "Run pipelines with unversioned config" option to **Off**.
```

**Review by rosieyohannan — APPROVED** [review]

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (BeFunes) [inline jekyll/_cci2/vs-code-extension-overview.adoc:216]: not sure if writing it this way is correct

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (BeFunes) [inline jekyll/_cci2/vs-code-extension-overview.adoc:204]: > See more details under Feature Settings.
 I wasn't sure how to add a link. @rosieyohannan can you help?

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (BeFunes) [inline jekyll/_cci2/vs-code-extension-overview.adoc:223]: I'm not confident in any of this wording, if there is anything you'd like to change please go ahead!

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (rosieyohannan) [inline jekyll/_cci2/vs-code-extension-overview.adoc:198]:
```suggestion
Use caution when running pipelines with _unversioned_ configuration. Take care to ensure no jobs with unintended effects are included in the run.
```

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (rosieyohannan) [inline jekyll/_cci2/vs-code-extension-overview.adoc:200]:
```suggestion
Test run pipelines triggered from VS Code have access to contexts and project environment variables based on branch and author permissions. We recommend using this feature **only** on feature branches.
```

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (rosieyohannan) [inline jekyll/_cci2/vs-code-extension-overview.adoc:202]:
```suggestion
The ability to trigger pipelines from VS Code is enabled by default on all projects. You can opt-out of this feature per project. For more details, see <<feature-controls>>. 
```

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (rosieyohannan) [inline jekyll/_cci2/vs-code-extension-overview.adoc:206]:
```suggestion
- Pipelines run with unversioned configuration are not differentiated from other pipelines in the CircleCI web app.
```

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (rosieyohannan) [inline jekyll/_cci2/vs-code-extension-overview.adoc:204]:
```suggestion
The following issue fixes are in development: 
```

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (rosieyohannan) [inline jekyll/_cci2/vs-code-extension-overview.adoc:227]: This would be better in the banner of things that are being worked on further up the page. We avoid time sensitive information outside banners to avoid outdated info remaining in the main text

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (rosieyohannan) [inline jekyll/_cci2/vs-code-extension-overview.adoc:223]:
```suggestion
The ability to trigger pipelines from VS Code is enabled by default on all projects. You can opt-out of this feature **at the project level**, as follows:
```

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (rosieyohannan) [inline jekyll/_cci2/vs-code-extension-overview.adoc:225]:
```suggestion
. Navigate to Project Settings[VS Code integration]
. Toggle the "Run pipelines with unversioned config" option to **Off** to opt-out
```

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (BeFunes) [reply jekyll/_cci2/vs-code-extension-overview.adoc:227]: How can we achieve this banner? 
To be clear, we will DEFINITELY remove this because it's very much top of mind at the moment.

Comment (rosieyohannan) [reply jekyll/_cci2/vs-code-extension-overview.adoc:228]: OK if it's very short term then that's fine

Comment (BeFunes) [reply jekyll/_cci2/vs-code-extension-overview.adoc]: Turned it into an info banner

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (rosieyohannan) [inline jekyll/_cci2/vs-code-extension-overview.adoc:222]: @BeFunes So is this a setting specific to the ability to use the feature through the VS Code extension, or will this also affect the ability to trigger with unversioned config via the API? If it's for both it seems like the menu position should be more general, if that makes sense?

Comment (BeFunes) [reply jekyll/_cci2/vs-code-extension-overview.adoc:222]: It's for both. 
Yes, it makes sense. We decided to put it in "VS Code Integrations" because at the moment it's the only place where we make this feature evident, and we thought this is the place where users would most likely look.
The "Advanced" page of proejct settings is so busy that we worried this wouldn't have been found. 

That said, I agree this should not be the permanent location of this setting, if Open Preview is validated and we decide to move forward and document it through the API etc

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (BeFunes) [inline jekyll/_cci2/vs-code-extension-overview.adoc:274]: Can we make this a smaller header? The section below still belongs to "Auditability"

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (BeFunes) [inline jekyll/_cci2/vs-code-extension-overview.adoc:191]: Can we extend it to this?
```
* Your org must have opted-in to this feature through Organization Settings. You may need to ask your org admin to do so for you. See <<feature-controls>> for more details.
```

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (BeFunes) [inline jekyll/_cci2/vs-code-extension-overview.adoc:189]: I missed this, it's now actually `v.2.3.0` or higher

**Review by BeFunes — COMMENTED** [review]
Added a couple of small comments but otherwise this is great!

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (BeFunes) [inline jekyll/_cci2/vs-code-extension-overview.adoc:282]: @JulesFaucherre will this be done and validated by Thursday? 
I don't mind leaving this "coming soon" and adjusting it at a later stage. Would give us a buffer for checking if all is good on the audit side. 
But I'll let you decide, you know how close we are to merging and testing in prod if audits have all they need

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (rosieyohannan) [inline jekyll/_cci2/vs-code-extension-overview.adoc:189]:
```suggestion
* CircleCI VS Code extension v2.3.0 or higher
```

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (rosieyohannan) [inline jekyll/_cci2/vs-code-extension-overview.adoc:191]:
```suggestion
* Your org must have opted-in to this feature through Organization Settings. You may need to ask your org admin to do so for you. See <<feature-controls>> for more details.
```

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (rosieyohannan) [inline jekyll/_cci2/vs-code-extension-overview.adoc:274]:
```suggestion
===== Coming soon
```

File: jekyll/_cci2/vs-code-extension-overview.adoc
Comment (JulesFaucherre) [reply jekyll/_cci2/vs-code-extension-overview.adoc:282]: The definition of the audit log can be found [here](https://github.com/circleci/circle/blob/0cb71fc721568e3feaabc9d7c6b989d1c55b0bac/src/circle/services/builds.clj#L400-L407). In the log you have:
 - Pipeline ID
 - Actor ID
 - Project ID
 - Branch or tag
 - Config source; value can be `"api"` or `"vcs"` where `"api"` means the config is unversioned because not coming from the VCS.

Comment (rosieyohannan) [reply jekyll/_cci2/vs-code-extension-overview.adoc:282]: @BeFunes I think this list needs updating in the doc before you merge, let me know and I will do it

**Pull request merged**
